common/src/test/java/com/hedera/block/common/utils/FileUtilitiesTest.java

```diff
@@ -32,9 +32,10 @@
 import org.junit.jupiter.params.provider.MethodSource;
 
 class FileUtilitiesTest {
+    private static final String FILE_WITH_UNRECOGNIZED_EXTENSION = "src/test/resources/nonexistent.unrecognized";
+
     @Test
-    void test_createPathIfNotExists_CreatesDirIfDoesNotExist(@TempDir final Path tempDir)
-            throws IOException {
+    void test_createPathIfNotExists_CreatesDirIfDoesNotExist(@TempDir final Path tempDir) throws IOException {
```

Comment:
Minor style note, we prefer to _not_ use mixed_SnakeCase like this for test names, and we also prefer to _not_ describe the test case in the method name with such detail.  It creates a lot of unnecessary maintenance with test method names changing for every test detail change.
Less detailed descriptions also work better with _parameterized_ tests (which are highly recommended).

Additional note:
This, and several following tests, could be a single parameterized `testCreatePathIfNotExists` and all the cases handled via parameter sets (likely from a `methodSource`).

Reply:
Noted, can I address this comment in #272 since both are related and in any case there are merge conflicts between them, no matter which one goes in first? I suggest that this one goes in first since let's say it is more or less resolved and is a lot simpler than #272 and in #272 I will resolve the merge conflicts and address this. Also in #272 there are additional changes to these tests and it would be more difficult to resolve merge conflicts if I change now all the names here as well. Sounds good?

Reply:
✅ Done. In addition, made minor changes to `StringUtilitiesTest` as well since I used the same naming conventions there.